Library/ViewModels/PledgeViewModel.swift

```diff
@@ -64,7 +64,7 @@ public protocol PledgeViewModelInputs {
   func riskMessagingViewControllerDismissed(isApplePay: Bool)
   func scaFlowCompleted(with result: StripePaymentHandlerActionStatusType, error: Error?)
   func shippingRuleSelected(_ shippingRule: ShippingRule)
-  func storeFacebookCAPIUserEmail(_ email: String?)
+
```

Comment:
can probably remove this newline, format might do it, if not don't worry about it.